src/components/Modals/NewCourse/SelectedRequirementEditor.vue

```diff
@@ -1,33 +1,56 @@
 <template>
   <div>
-    <div v-if="requirementsThatAllowDoubleCounting.length > 0">
-      <div class="newCourse-title">
-        This class automatically fulfills the following requirement(s):
```

Comment:
i'm sorry sam, could we add the "automatically" back on the second page? looking now, i think it mights sense to have it for consistency. sorry!